tests/ServerTest.php

```diff
@@ -81,8 +81,9 @@ public function testParserErrorEmitted()
         $conn = new ConnectionStub();
         $io->emit('connection', [$conn]);
 
-        $data = $this->createGetRequest();
-        $data = str_pad($data, 4096 * 4);
+        $data = chop($this->createGetRequest());
```

Comment:
This looks *okay*, but could probably be a bit *clearer*. What do you think, should this be updated to use a literal request string here instead?